unresolved imports `mopro_ffi::ProofCalldata`

# Problem
A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

```sh
error[E0432]: unresolved imports `mopro_ffi::ProofCalldata`, `mopro_ffi::G1`, `mopro_ffi::G2`
  --> src/lib.rs:9:1
   |
9  | mopro_ffi::app!();
   | ^^^^^^^^^^^^^^^^^
   | |
   | no `ProofCalldata` in the root
   | no `G1` in the root
   | no `G2` in the root
   |
note: found an item that was configured out
  --> /Users/zhengyawen/.cargo/git/checkouts/mopro-af576ae5c07fa51d/55a0790/mopro-ffi/src/lib.rs:11:5
   |
11 |     ProofCalldata, G1, G2,
   |     ^^^^^^^^^^^^^
note: the item is gated behind the `circom` feature
  --> /Users/zhengyawen/.cargo/git/checkouts/mopro-af576ae5c07fa51d/55a0790/mopro-ffi/src/lib.rs:8:7
   |
8  | #[cfg(feature = "circom")]
   |       ^^^^^^^^^^^^^^^^^^
note: found an item that was configured out
  --> /Users/zhengyawen/.cargo/git/checkouts/mopro-af576ae5c07fa51d/55a0790/mopro-ffi/src/lib.rs:11:20
   |
11 |     ProofCalldata, G1, G2,
   |                    ^^
note: the item is gated behind the `circom` feature
  --> /Users/zhengyawen/.cargo/git/checkouts/mopro-af576ae5c07fa51d/55a0790/mopro-ffi/src/lib.rs:8:7
   |
8  | #[cfg(feature = "circom")]
   |       ^^^^^^^^^^^^^^^^^^
note: found an item that was configured out
  --> /Users/zhengyawen/.cargo/git/checkouts/mopro-af576ae5c07fa51d/55a0790/mopro-ffi/src/lib.rs:11:24
   |
11 |     ProofCalldata, G1, G2,
   |                        ^^
note: the item is gated behind the `circom` feature
  --> /Users/zhengyawen/.cargo/git/checkouts/mopro-af576ae5c07fa51d/55a0790/mopro-ffi/src/lib.rs:8:7
   |
8  | #[cfg(feature = "circom")]
   |       ^^^^^^^^^^^^^^^^^^
   = note: this error originates in the macro `mopro_ffi::app` (in Nightly builds, run with -Z macro-backtrace for more info)
```

## Details
When running `mopro-example-app`, and the `mopro-ffi/circom` is disabled
then the structures cannot be found with the fixed UDL file

## Acceptance criteria
Should be fixed by #306

--- a/mopro-ffi/src/lib.rs
+++ b/mopro-ffi/src/lib.rs
@@ -86,6 +86,28 @@ pub struct GenerateProofResult {
     pub inputs: Vec<u8>,
 }
 
+#[cfg(not(feature = "circom"))]
+#[derive(Debug, Clone, Default)]
+pub struct G1 {
+    pub x: String,
+    pub y: String,
+}
+
+#[cfg(not(feature = "circom"))]
+#[derive(Debug, Clone, Default)]
+pub struct G2 {
+    pub x: Vec<String>,
+    pub y: Vec<String>,
+}
+
+#[cfg(not(feature = "circom"))]
+#[derive(Debug, Clone, Default)]
+pub struct ProofCalldata {
+    pub a: G1,
+    pub b: G2,
+    pub c: G1,
+}
+
 /// This macro is used to setup the Mopro FFI library
 /// It should be included in the `lib.rs` file of the project
 ///

